### PR TITLE
PIL-815: permissions screen refactor

### PR DESCRIPTION
**Pull request by poocart**
https://linear.app/pillarproject/issue/PIL-815/refactor-permissions-screen

Permissions screen refactor with hooks and function component.

### CODE REVIEW

File: src/screens/Permissions/Permissions.js [hunk 1]
```diff
@@ -92,14 +94,16 @@ const TickIcon = styled(Icon)`
...
+const Permissions = ({
+  navigation,
+  theme,
+}: Props) => {
+  const [openCollapseKey, setOpenCollapseKey] = useState(null);
+  const [openInnerCollapseKey, setOpenInnerCollapseKey] = useState(null);
+  const [hasAgreedToTerms, setHasAgreedToTerms] = useState(false);
 
-  sections = [
+  // keep sections in component scope due translations
```
Comment (vyvil) [inline src/screens/Permissions/Permissions.js:105]: 👍

**Review by vyvil — APPROVED** [review]
Neat :)

**Review by graszka22 — COMMENTED** [review]
Just a little bug probably with translations:

Screenshot 2020-08-27 at 15 02 41

**Comment by vyvil** [comment]
@graszka22, yeah. Noticed that too... Adding a fix into my latest PR :)

**Review by graszka22 — APPROVED** [review]

**Pull request merged**